.coderabbit.yaml

```
@@ -1,201 +1,68 @@
-    static_analysis: true
-
-# 学習とカスタマイズ
-learning:
-  # プロジェクト固有のパターンを学習
-  learn_project_patterns: true
-  # フィードバックから学習
-  learn_from_feedback: true
-  # 継続的改善
-  continuous_improvement: true 
\ No newline at end of file
+  safe_only: true 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)